CMakeLists.txt

```diff
@@ -52,6 +52,22 @@ if (NOT BUILD_NUMBER)
   set(BUILD_NUMBER "0")
 endif ()
 
+if (UNIX)
+  option(BUILD_ENCLAVES "Build Enclaves" ON)
+else()
```

Comment:
I'd actually swap this logic to specifically be `if (WIN32) ... else () # everything else`. Since it's Windows that we're special-casing, and there is more out than just Windows and Unix.

Reply:
Fixed.